Xamarin.Forms.Platform.Android/VisualElementPackager.cs

```diff
@@ -66,6 +66,9 @@ protected virtual void Dispose(bool disposing)
 				}
 				_renderer = null;
 			}
+
+			if (_element != null)
```

Comment:
Why check? Just null it

Reply:
you are right , is was doing some unsubscribe and then i removed that code..

Reply:
will fix before merge..